ocs_ci/ocs/resources/mcg.py

```diff
@@ -799,20 +799,41 @@ def _check_mirroring():
                     else:
                         results.append(False)
 
-            return all(results)
-
-        try:
-            for mirroring_is_complete in TimeoutSampler(timeout, 5, _check_mirroring):
-                if mirroring_is_complete:
+            return results
+
+        is_mirroring_complete = _check_mirroring()
+        current_percentage = (
+            is_mirroring_complete.count(True) / len(is_mirroring_complete)
+        ) * 100
+        logger.info(f"{current_percentage}% mirroring is done.")
+        previous_percentage = 0
+        if current_percentage == 100:
+            logger.info("All objects mirrored successfully.")
+            return True
+        else:
+            while current_percentage < 100:
+                previous_percentage = current_percentage
+                for i in range(timeout):
+                    is_mirroring_complete = _check_mirroring()
+                    current_percentage = (
+                        is_mirroring_complete.count(True) / len(is_mirroring_complete)
+                    ) * 100
+                    if previous_percentage == current_percentage:
+                        import time
```

Comment:
Imports should always be at the top of the file. Here are two reasons why this matters:

1.  Readability: we should strive to keep our code succinct to reduce the amount of time it takes to read and understand it. The use of the time module is obvious in line 824 so there's no need to include this import here - no new information of functionality is being added.

2. Unnecessary overhead that (slightly) affects performance.

Reply:
Also, I see that time.sleep is already imported at line 9 so you should just use `sleep` instead of `time.sleep`:
```
from time import sleep
```

Reply:
Fixed